OutOfSchool/OutOfSchool.DataAccess/Models/DateTimeRange.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using OutOfSchool.Services.Enums;
+
+namespace OutOfSchool.Services.Models
+{
+    public class DateTimeRange
+    {
+        public long Id { get; set; }
+
+        public TimeSpan StartTime { get; set; }
+
+        public TimeSpan EndTime { get; set; }
+
+        public long WorkshopId { get; set; }
+
+        public List<DaysBitMask> Workdays { get; set; }
```

Comment:
Why list?

Reply:
It can be serialized and deserialized using default Newtonsoft list converter

Reply:
Resolved after discussion that code changes requires significant amount of extra code and extra dependency added